src/Bridge/Symfony/Bundle/Test/Client.php

```diff
@@ -212,4 +212,24 @@ public function enableReboot(): void
     {
         $this->kernelBrowser->enableReboot();
     }
+
+    /**
+     * Normalize headers depending on the symfony/http-client version being used.
+     *
+     * @return array<string, string[]>
+     */
+    private static function extractHeaders(array $options): array
+    {
+        $headers = $options['normalized_headers'] ?? $options['headers'];
+        foreach ($headers as $key => &$values) {
+            foreach ($values as &$value) {
+                $prefix = $key.':';
+                if (0 === stripos($value, $prefix)) {
+                    $value = ltrim(substr($value, \strlen($prefix)));
```

Comment:
Hmm... This piece of code seems unnecessarily complex and error prone to me...

Reply:
Passing by reference == matter of taste, but I don't see any pb in removing them.
The goal is to remove the prefix from the value: prefix is key (original-cased), clon and space. Which approach do you recommend?